ui/component/walletSendTip/view.jsx

```diff
@@ -186,8 +186,7 @@ function WalletSendTip(props: Props) {
 
   React.useEffect(() => {
     // Regex for number up to 8 decimal places
-    const regexp = RegExp(/^(\d*([.]\d{0,8})?)$/);
-    const validTipInput = regexp.test(String(tipAmount));
+    let regexp;
```

Comment:
Two separate well-labeled regexps FIAT_REGEX, CRYPTO_REGEX at the top of the file?